### ISSUE TITLE
Bug/Upgrade from crowdsec 1.1.1 -> 1.2.1 fails on Debian

### ISSUE THREAD

**Issue by klausagnoletti** [crowdsecurity/crowdsec#1111]
Please, start your issue name (after `Bug`) with the component name impacted by this feature request and a small description of the Bug. Example: `Bug/cscli: issue with ....` and remove this line :)

**Describe the bug**
On a clean install of Debian 10.11 I first installed crowdsec 1.1.1 and iptables bouncer 0.0.15 from debs. Next I upgraded. That failed because crowdsec failed to start after upgrade. 
```
time="17-12-2021 17:27:22" level=error msg="failed to load pattern mysql.dpkg-dist : the pattern already exist"
time="17-12-2021 17:27:22" level=fatal msg="crowdsec init: Failed to load parsers: failed to load parser patterns : the pattern already exist"
```
This was in the logs. After deleting the mysql.dpkg-dist in /etc/crowdsec/patterns I was able to start the crowdsec agent and finish the upgrade.

**To Reproduce**
1. Install Debian 10.11
2. Add package cloud repo
3. install crowdsec=1.1.1 and crowdsec-firewall-iptables=0.1.5
4. apt upgrade so latest version of both packages are installed

**Expected behavior**
I expected upgrade to not fail :-)

**Technical Information (please complete the following information):**
 - OS: Debian
 - Version 10.11

**Comment by thib3113** [comment]
Go in `/etc/crowdsec/parsers/`, you will maybe got some patterns in double (and some with `.dpkg-dist` in the name)

**Comment by interzone2012** [comment]
Op vr 17 dec. 2021 21:44 schreef Thibaut SEVERAC ***@***.***>:

**Comment by cronlabspl** [comment]
@klausagnoletti Update went well, its `crowdsec` thats failing

**Comment by thib3113** [comment]
1.2.2 seems to doesn't resolve the problem .

An example : 

The decision from the list 
```
$ cscli decisions list | grep 61.177.173.8
| 207315 | crowdsec | Ip:61.177.173.8    | crowdsecurity/ssh-slow-bf | ban    | CN      | 4134 No.31,Jin-rong Street     |     12 | 2h1m49.460367218s  |     5895 |
```

The logs from the firewall show some lines like : 
```
time="24-12-2021 13:30:27" level=info msg="Error while inserting in set (/sbin/ipset -exist add crowdsec-blacklists 61.177.173.8 timeout -50): exit status 1 --> ipset v7.5: Syntax error: '-50' is out of range 0-2147483\n"
```

Another strange things, it's that I have really lot of lines where this ip is banned .
( part of the return from `cat /var/log/crowdsec-firewall-bouncer.log | grep 61.177.173.` )
```
time="24-12-2021 13:30:27" level=debug msg="ipset add ban [61.177.173.8] (for -3309 seconds)"
time="24-12-2021 13:30:27" level=debug msg="ipset add : /sbin/ipset -exist add crowdsec-blacklists 61.177.173.8 timeout -3309"
time="24-12-2021 13:30:27" level=info msg="Error while inserting in set (/sbin/ipset -exist add crowdsec-blacklists 61.177.173.8 timeout -3309): exit status 1 --> ipset v7.5: Syntax error: '-3309' is out of range 0-2147483\n"
time="24-12-2021 13:30:27" level=debug msg="Adding '61.177.173.8' for '-55m9.243834904s'"
time="24-12-2021 13:30:27" level=debug msg="ipset add ban [61.177.173.8] (for -3099 seconds)"
time="24-12-2021 13:30:27" level=debug msg="ipset add : /sbin/ipset -exist add crowdsec-blacklists 61.177.173.8 timeout -3099"
time="24-12-2021 13:30:27" level=info msg="Error while inserting in set (/sbin/ipset -exist add crowdsec-blacklists 61.177.173.8 timeout -3099): exit status 1 --> ipset v7.5: Syntax error: '-3099' is out of range 0-2147483\n"
time="24-12-2021 13:30:27" level=debug msg="Adding '61.177.173.8' for '-51m39.711244736s'"
time="24-12-2021 13:30:27" level=debug msg="ipset add ban [61.177.173.8] (for -2712 seconds)"
time="24-12-2021 13:30:27" level=debug msg="ipset add : /sbin/ipset -exist add crowdsec-blacklists 61.177.173.8 timeout -2712"
time="24-12-2021 13:30:27" level=info msg="Error while inserting in set (/sbin/ipset -exist add crowdsec-blacklists 61.177.173.8 timeout -2712): exit status 1 --> ipset v7.5: Syntax error: '-2712' is out of range 0-2147483\n"
time="24-12-2021 13:30:27" level=debug msg="Adding '61.177.173.8' for '-45m12.381012751s'"
time="24-12-2021 13:30:27" level=debug msg="ipset add ban [61.177.173.8] (for -2328 seconds)"
time="24-12-2021 13:30:27" level=debug msg="ipset add : /sbin/ipset -exist add crowdsec-blacklists 61.177.173.8 timeout -2328"
time="24-12-2021 13:30:27" level=info msg="Error while inserting in set (/sbin/ipset -exist add crowdsec-blacklists 61.177.173.8 timeout -2328): exit status 1 --> ipset v7.5: Syntax error: '-2328' is out of range 0-2147483\n"
time="24-12-2021 13:30:27" level=debug msg="Adding '61.177.173.8' for '-38m48.014345357s'"
time="24-12-2021 13:30:27" level=debug msg="ipset add ban [61.177.173.8] (for -2234 seconds)"
time="24-12-2021 13:30:27" level=debug msg="ipset add : /sbin/ipset -exist add crowdsec-blacklists 61.177.173.8 timeout -2234"
time="24-12-2021 13:30:27" level=info msg="Error while inserting in set (/sbin/ipset -exist add crowdsec-blacklists 61.177.173.8 timeout -2234): exit status 1 --> ipset v7.5: Syntax error: '-2234' is out of range 0-2147483\n"
time="24-12-2021 13:30:27" level=debug msg="Adding '61.177.173.8' for '-37m14.724052996s'"
time="24-12-2021 13:30:27" level=debug msg="ipset add ban [61.177.173.8] (for -2227 seconds)"
time="24-12-2021 13:30:27" level=debug msg="ipset add : /sbin/ipset -exist add crowdsec-blacklists 61.177.173.8 timeout -2227"
time="24-12-2021 13:30:27" level=info msg="Error while inserting in set (/sbin/ipset -exist add crowdsec-blacklists 61.177.173.8 timeout -2227): exit status 1 --> ipset v7.5: Syntax error: '-2227' is out of range 0-2147483\n"
time="24-12-2021 13:30:27" level=debug msg="Adding '61.177.173.8' for '-37m7.957327771s'"
time="24-12-2021 13:30:27" level=debug msg="ipset add ban [61.177.173.8] (for -1431 seconds)"
time="24-12-2021 13:30:27" level=debug msg="ipset add : /sbin/ipset -exist add crowdsec-blacklists 61.177.173.8 timeout -1431"
time="24-12-2021 13:30:27" level=info msg="Error while inserting in set (/sbin/ipset -exist add crowdsec-blacklists 61.177.173.8 timeout -1431): exit status 1 --> ipset v7.5: Syntax error: '-1431' is out of range 0-2147483\n"
time="24-12-2021 13:30:27" level=debug msg="Adding '61.177.173.8' for '-23m51.073589152s'"
time="24-12-2021 13:30:27" level=debug msg="ipset add ban [61.177.173.8] (for -1239 seconds)"
time="24-12-2021 13:30:27" level=debug msg="ipset add : /sbin/ipset -exist add crowdsec-blacklists 61.177.173.8 timeout -1239"
time="24-12-2021 13:30:27" level=info msg="Error while inserting in set (/sbin/ipset -exist add crowdsec-blacklists 61.177.173.8 timeout -1239): exit status 1 --> ipset v7.5: Syntax error: '-1239' is out of range 0-2147483\n"
time="24-12-2021 13:30:27" level=debug msg="Adding '61.177.173.8' for '-20m39.331262282s'"
time="24-12-2021 13:30:27" level=debug msg="ipset add ban [61.177.173.8] (for -1239 seconds)"
time="24-12-2021 13:30:27" level=debug msg="ipset add : /sbin/ipset -exist add crowdsec-blacklists 61.177.173.8 timeout -1239"
time="24-12-2021 13:30:27" level=info msg="Error while inserting in set (/sbin/ipset -exist add crowdsec-blacklists 61.177.173.8 timeout -1239): exit status 1 --> ipset v7.5: Syntax error: '-1239' is out of range 0-2147483\n"
time="24-12-2021 13:30:27" level=debug msg="Adding '61.177.173.8' for '-20m39.331083638s'"
time="24-12-2021 13:30:27" level=debug msg="ipset add ban [61.177.173.8] (for -146 seconds)"
time="24-12-2021 13:30:27" level=debug msg="ipset add : /sbin/ipset -exist add crowdsec-blacklists 61.177.173.8 timeout -146"
time="24-12-2021 13:30:27" level=info msg="Error while inserting in set (/sbin/ipset -exist add crowdsec-blacklists 61.177.173.8 timeout -146): exit status 1 --> ipset v7.5: Syntax error: '-146' is out of range 0-2147483\n"
time="24-12-2021 13:30:27" level=debug msg="Adding '61.177.173.8' for '-2m26.718715373s'"
time="24-12-2021 13:30:27" level=debug msg="ipset add ban [61.177.173.8] (for -50 seconds)"
time="24-12-2021 13:30:27" level=debug msg="ipset add : /sbin/ipset -exist add crowdsec-blacklists 61.177.173.8 timeout -50"
time="24-12-2021 13:30:27" level=info msg="Error while inserting in set (/sbin/ipset -exist add crowdsec-blacklists 61.177.173.8 timeout -50): exit status 1 --> ipset v7.5: Syntax error: '-50' is out of range 0-2147483\n"
time="24-12-2021 13:30:27" level=debug msg="Adding '61.177.173.8' for '-50.893422292s'"
```

Also, if I search `cat /var/log/crowdsec-firewall-bouncer.log | grep "deleted '61.177.173.8'"` . 
I have 343 lines, between `13:30:14` and `13:30:27` (just after restarting the service) (the service try to delete this ip 343 times ?)

I saw that too : 
```
cat /var/log/crowdsec-firewall-bouncer.log | grep "Adding '61.177.173.8'"
time="24-12-2021 13:30:27" level=debug msg="Adding '61.177.173.8' for '-55m9.243834904s'"
time="24-12-2021 13:30:27" level=debug msg="Adding '61.177.173.8' for '-51m39.711244736s'"
time="24-12-2021 13:30:27" level=debug msg="Adding '61.177.173.8' for '-45m12.381012751s'"
time="24-12-2021 13:30:27" level=debug msg="Adding '61.177.173.8' for '-38m48.014345357s'"
time="24-12-2021 13:30:27" level=debug msg="Adding '61.177.173.8' for '-37m14.724052996s'"
time="24-12-2021 13:30:27" level=debug msg="Adding '61.177.173.8' for '-37m7.957327771s'"
time="24-12-2021 13:30:27" level=debug msg="Adding '61.177.173.8' for '-23m51.073589152s'"
time="24-12-2021 13:30:27" level=debug msg="Adding '61.177.173.8' for '-20m39.331262282s'"
time="24-12-2021 13:30:27" level=debug msg="Adding '61.177.173.8' for '-20m39.331083638s'"
time="24-12-2021 13:30:27" level=debug msg="Adding '61.177.173.8' for '-2m26.718715373s'"
time="24-12-2021 13:30:27" level=debug msg="Adding '61.177.173.8' for '-50.893422292s'"
time="24-12-2021 13:30:27" level=debug msg="Adding '61.177.173.8' for '48.354991463s'"
time="24-12-2021 13:30:27" level=debug msg="Adding '61.177.173.8' for '5m32.543940379s'"
time="24-12-2021 13:30:27" level=debug msg="Adding '61.177.173.8' for '22m1.626325385s'"
time="24-12-2021 13:30:27" level=debug msg="Adding '61.177.173.8' for '23m48.226649618s'"
time="24-12-2021 13:30:27" level=debug msg="Adding '61.177.173.8' for '46m35.436611466s'"
time="24-12-2021 13:30:27" level=debug msg="Adding '61.177.173.8' for '53m9.186354122s'"
time="24-12-2021 13:30:27" level=debug msg="Adding '61.177.173.8' for '53m9.298172412s'"
time="24-12-2021 13:30:27" level=debug msg="Adding '61.177.173.8' for '54m46.559951991s'"
time="24-12-2021 13:30:27" level=debug msg="Adding '61.177.173.8' for '1h12m57.281069406s'"
time="24-12-2021 13:30:27" level=debug msg="Adding '61.177.173.8' for '1h17m55.553882356s'"
time="24-12-2021 13:30:27" level=debug msg="Adding '61.177.173.8' for '1h32m32.56110653s'"
time="24-12-2021 13:30:27" level=debug msg="Adding '61.177.173.8' for '1h35m55.510516687s'"
time="24-12-2021 13:30:27" level=debug msg="Adding '61.177.173.8' for '1h35m55.555359698s'"
time="24-12-2021 13:30:27" level=debug msg="Adding '61.177.173.8' for '1h39m14.525419604s'"
time="24-12-2021 13:30:27" level=debug msg="Adding '61.177.173.8' for '1h42m31.014300607s'"
time="24-12-2021 13:30:27" level=debug msg="Adding '61.177.173.8' for '1h47m27.283383313s'"
time="24-12-2021 13:30:27" level=debug msg="Adding '61.177.173.8' for '1h50m29.500308923s'"
time="24-12-2021 13:30:27" level=debug msg="Adding '61.177.173.8' for '1h50m35.259987208s'"
time="24-12-2021 13:30:27" level=debug msg="Adding '61.177.173.8' for '1h52m15.022021156s'"
time="24-12-2021 13:30:27" level=debug msg="Adding '61.177.173.8' for '2h3m47.104550148s'"
time="24-12-2021 13:30:27" level=debug msg="Adding '61.177.173.8' for '2h5m28.115773049s'"
time="24-12-2021 13:30:27" level=debug msg="Adding '61.177.173.8' for '2h5m28.115921098s'"
time="24-12-2021 13:30:27" level=debug msg="Adding '61.177.173.8' for '2h15m11.849357898s'"
time="24-12-2021 13:30:27" level=debug msg="Adding '61.177.173.8' for '2h20m4.111531028s'"
time="24-12-2021 13:30:27" level=debug msg="Adding '61.177.173.8' for '2h44m47.589758681s'"
time="24-12-2021 13:30:27" level=debug msg="Adding '61.177.173.8' for '2h49m54.127803487s'"
time="24-12-2021 13:30:27" level=debug msg="Adding '61.177.173.8' for '3h4m43.002541778s'"
time="24-12-2021 13:30:27" level=debug msg="Adding '61.177.173.8' for '3h4m43.002791665s'"
time="24-12-2021 13:30:27" level=debug msg="Adding '61.177.173.8' for '3h7m57.03650689s'"
time="24-12-2021 13:30:27" level=debug msg="Adding '61.177.173.8' for '3h11m14.799344426s'"
time="24-12-2021 13:30:27" level=debug msg="Adding '61.177.173.8' for '3h31m7.935390647s'"
time="24-12-2021 13:30:27" level=debug msg="Adding '61.177.173.8' for '3h31m7.935551097s'"
time="24-12-2021 13:30:34" level=debug msg="Adding '61.177.173.8' for '3h59m6.553842459s'"
[logs below arrive when I write this comment]
time="24-12-2021 13:36:14" level=debug msg="Adding '61.177.173.8' for '3h59m56.664836454s'"
time="24-12-2021 13:39:34" level=debug msg="Adding '61.177.173.8' for '3h59m53.92548068s'"
```

( I'm using v1.2.2 from 48h+, and bantime is 24h, -55m seems to doesn't come from the old version )


---
After restarting the service, 61.177.173.8 is in the ipset list, but before, it is not

**Comment by Athanasius** [comment]
I too encountered this.  `/etc/crowdsec/parsers` contained no duplicates (checked with `ls -alR` in there).

TL;DR - Likely something to do with leapfrogging so many versions *or* the .dpkg file not ensuring it definitely updates all the relevant files to their latest versions.

In the end I:

1. backed up the two credentials files (I had no custom config, and all of /etc is under git anyway).
2. `apt purge crowdsec`
3. `apt install crowdsec` (which made new credentials files anyway)

and it worked.  Now, I was coming from some ancient 1.0.2 version, because no hint was given that the APT repository had changed, so it's possible that necessary tweaks to configuration/other files were missed due to that leap.

I note that performing that upgrade did **not** change the files under `/etc/crowdsec/parsers` (which are of course symlinks) *or the files those symlinks point to*, **but the purge/install cycle has**.  So something definitely went awry with the APT upgrade of the package.

I had `/etc/` committed and in a clean state before the purge/install, but after the upgrade from 1.0.2 to 1.3.1.  Now I've done the purge/install cycle I'm showing differences in, e.g. `crowdsec/hub/parsers/s00-raw/crowdsecurity/syslog-logs.yaml` in a `git diff`.  I've now commited those changes, and for my own reference that's local /etc commit `c165f4aea87d68f530dad5f285e85d5aa6bfba8`.
